src/AElf.Kernel.Core/SmartContract/Application/IBlockchainStateService.cs

```diff
@@ -16,6 +18,15 @@ public interface IBlockchainStateService
         Task SetBlockStateSetAsync(BlockStateSet blockStateSet);
 
         Task RemoveBlockStateSetsAsync(IList<Hash> blockStateHashes);
+        Task<TEntity> GetBlockExecutedDataAsync<TEntity>(IChainContext chainContext);
+
+        Task<TEntity> GetBlockExecutedDataAsync<TKey, TEntity>(IChainContext chainContext, TKey key);
+        
+        Task AddBlockExecutedDataAsync<TEntity>(Hash blockHash, TEntity blockExecutedData);
+        
+        Task AddBlockExecutedDataAsync<TKey, TEntity>(Hash blockHash, TKey key, TEntity value);
+        
+        Task AddBlockExecutedDataAsync<TKey, TEntity>(Hash blockHash, IDictionary<TKey, TEntity> blockExecutedData);
```

Comment:
only keep one Task AddBlockExecutedDataAsync<TEntity>(Hash blockHash, TEntity blockExecutedData);

the other make in extension method